primes.rb

```diff
@@ -1,6 +1,16 @@
+require 'prime'
+
 class Primes
-  def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
+  def self.sum_to(num)
+    Prime.each(num).inject(&:+)
```

Comment:
How's this? Seems to work OK: 

def prime_nums(max_num)
numbers = Array(2..max_num)
p = Array.new

	while numbers.length > 0 do
	p.push(numbers.first)
	numbers.delete_if { |x| x % p.last == 0 }
	end

	puts p

end

prime_nums(100)